src/MSRIOGroup.cpp

```diff
@@ -687,7 +692,11 @@ namespace geopm
         std::string result = "Invalid signal description: no description found.";
         auto it = m_signal_available.find(signal_name);
         if (it != m_signal_available.end()) {
-            result = it->second.description;
+            result =  "    description: " + it->second.description + '\n'; // Includes alias_for if applicable
```

Comment:
We plan on reimplementing this with the function that takes a structure, but when we do, we should use std::ostringstream for any operations like this.